packages/content/types/database.d.ts

```diff
@@ -0,0 +1,108 @@
+import type { Loki, Collection } from '@lokidb/loki';
+import type { Queue, DefaultAddOptions } from 'p-queue';
+import type { FSWatcher } from 'chokidar'
+import type PriorityQueue from 'p-queue/dist/priority-queue';
+import type { State as YamlOptions } from 'js-yaml';
+import type { Options as XMLOptions } from 'xml2js';
+import type { CSVParseParam as CSVOptions } from 'csvtojson/v2/Parameters';
+
+import type { IMarkdown } from './parsers/markdown';
+import type { IYAML } from './parsers/yaml';
+import type { ICSV } from './parsers/csv';
+import type { IXML } from './parsers/xml';
+import type { MarkdownOptions } from './parsers/markdown';
+import type { QueryBuilder } from './query-builder'
+import { IContentDocument } from './content';
```

Comment:
Only regular import, can be `import type` as well I think?

Reply:
According to this doc https://www.typescriptlang.org/docs/handbook/modules.html#importing-types, `import type` should be worked.